### ISSUE TITLE
predict the accuracy for the mask for every class in multiple class

### ISSUE THREAD

**Issue by Mahmuod1** [qubvel/segmentation_models.pytorch#660]
how i can make the model predict also the accuracy of the mask for every class in multiple class semantic segmenation

**Comment by H1ghSpeed** [comment]
Hello, show me your code.
I did it like this:

```
iou = torch.zeros(self.classes_num)
tp, fp, fn, tn = smp.metrics.get_stats(output, target, mode='multilabel', threshold=0.5)

for batch in range(input.shape[0]):
    for num_class in range(self.classes_num):
        iou[i] = smp.metrics.iou_score(tp[batch][num_class], fp[batch][num_class], fn[batch][num_class], tn[batch][num_class], reduction="macro")
```
